### PR TITLE
Adding 2.263.x line

### PR DESCRIPTION
**Pull request by jglick**
Presuming this is finalized as the next baseline. Requires https://github.com/jenkins-infra/repository-permissions-updater/pull/1730 to release.

### CODE REVIEW

**Comment by jglick** [comment]
```
java.lang.VerifyError: class org.jenkinsci.plugins.workflow.steps.CoreWrapperStepTest$WrapperWithoutWorkspaceRequirement$DisposerWithoutWorkspaceRequirement overrides final method requiresWorkspace.()Z
  at org.jenkinsci.plugins.workflow.steps.CoreWrapperStepTest$WrapperWithoutWorkspaceRequirement$DescriptorImpl.<init>(CoreWrapperStepTest.java:366)
```

**Comment by timja** [comment]
doesn't sound good

**Comment by timja** [comment]
turn fail fast off in case there's others?

File: failFast [hunk 1]
```diff
@@ -1 +1 @@
-true
+false
```
Comment (jglick) [inline failFast:1]: **revert**

**Comment by jglick** [comment]
I hope I do not need to revert #319 and restore the ugly throttling. Something keeps killing the agents here…

**Review by MRamonLeon — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Thanks @jglick !

**Pull request merged**